pkg/plugins/policies/core/rules/rules.go

```diff
@@ -44,7 +45,8 @@ func (i InboundListener) String() string {
 }
 
 type FromRules struct {
-	Rules map[InboundListener]Rules
+	Rules    map[InboundListener]Rules
+	RulesNew map[InboundListener][]*inbound.Rule
```

Comment:
can we try finding better name than `RulesNew`? What if we add new new rules in the future?

Reply:
Agreed, renamed to `InboundRule` and marked old `Rules` as deprecated